site/blog/unlisted/subqueries.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: subquery-vs-join
+title: Subquery vs JOIN
+authors: [depombo]
+tags: [sql]
+---
+
+As a developer, you are probably familiar with the concept of joins in SQL databases. Joins are a powerful tool for combining data from multiple tables, but there are certain situations where subqueries may be a better choice. In this blog post, we will look at the advantages of using subqueries instead of joins in PostgreSQL, and why it may be the better option for your project.
+
+### Improved Performance
+One of the biggest advantages of using subqueries is improved performance. Subqueries can be optimized and executed more efficiently than joins, especially when dealing with large amounts of data. In PostgreSQL, subqueries can be executed in parallel, which can significantly improve performance. By breaking down the query into smaller parts, the database can use its indexing and caching mechanisms to optimize the performance of the query. In contrast, joins can be slower and more resource-intensive, especially when dealing with complex queries or large datasets. 
```

Comment:
There should be a caveat here. The output of the sub-query itself will not be indexed, so the cardinality of the output needs to be much smaller than the inputs for the performance benefit to actually occur.